--- setup.py	Sun Oct 18 15:08:32 2015
+++ setup.py	Sun Oct 18 15:38:05 2015
@@ -19,9 +19,10 @@
         shutil.copytree('murmurhash/headers/murmurhash', dest_dir)
 
 def rm_cflag(text):
-    cflags = distutils.sysconfig._config_vars['CFLAGS']
-    cflags = cflags.replace(text, '')
-    distutils.sysconfig._config_vars['CFLAGS'] = cflags
+    cflags = distutils.sysconfig.get_config_var('CFLAGS')
+    if cflags is not None:
+        cflags = cflags.replace(text, '')
+        distutils.sysconfig._config_vars['CFLAGS'] = cflags
 
 install_headers()
 includes = ['.', path.join(sys.prefix, 'include')]
